optuna/_gp/optim_mixed.py

```diff
@@ -262,7 +262,7 @@ def local_search_mixed(
             # Parameters not changed from the beginning.
             return (best_normalized_params, best_fval)
 
-    _logger.warn("local_search_mixed: Local search did not converge.")
+    _logger.warning("local_search_mixed: Local search did not converge.")
```

Comment:
Could you make this line covered by tests?

Reply:
Sorry, I must have misunderstood, but this comment is outside the scope of the PR.